libsolidity/analysis/NameAndTypeResolver.h

```diff
@@ -87,6 +87,11 @@ class NameAndTypeResolver: private boost::noncopyable
 	/// Should only be called during the initial resolving phase.
 	std::vector<Declaration const*> nameFromCurrentScope(ASTString const& _name, bool _includeInvisibles = false) const;
 
+	/// Resolves a name in the "enclosing" scope, but also searches parent scopes.
```

Comment:
```suggestion
	/// Resolves a name in the enclosing or parent scopes.
```

Reply:
That sounds better for me too. However see other comments in the same header:
```
/// Resolves a name in the "current" scope, but also searches parent scopes.
/// Should only be called during the initial resolving phase.
std::vector<Declaration const*> nameFromCurrentScope(ASTString const& _name, bool _includeInvisibles = false) const;

/// Resolves a path starting from the "current" scope, but also searches parent scopes.
/// Should only be called during the initial resolving phase.
/// @note Returns a null pointer if any component in the path was not unique or not found.
Declaration const* pathFromCurrentScope(std::vector<ASTString> const& _path) const;
```

Should we change these comments altogether separately?